README.md

```diff
@@ -29,6 +29,13 @@ It's a simple demo app, which allows user to list categories, categories' offers
 	  id:  "3"
 	}]
 
+`GET /api/categories/{category_id}`
```

Comment:
To rozwiązanie może teraz zostać, ale pewnie trzeba będzie to jakoś inaczej rozwiązać, żeby przy wejściu na stronę ofert z danej kategorii nie uderzać do dwóch endpointów, coś się wymyśli ;)